### PR TITLE
Added txlSubscribe, openFile, createBucket + docs

### PR DESCRIPTION
**Pull request by gpuente**
## Changelog

- Added `txlSubscribe` method
- Added `openFile` method
- Added `createBucket` method
- Added docs for: `txlSubscribe`, `openFile`, `createBucket` and `listDirectories `

### CODE REVIEW

**Comment by gpuente** [comment]
[ch16938]
[ch16874]
[ch16868]

File: src/client.ts [hunk 1]
```diff
@@ -42,6 +48,60 @@ class SpaceClient {
...
+  txlSubscribe(
+    metadata: grpcWeb.Metadata = {},
+  ): grpcWeb.ClientReadableStream<TextileEventResponse> {
+    const request = new Empty();
+
+    return this.instance.txlSubscribe(request, metadata);
+  }
+
+  openFile(
+    path: string,
```
Comment (giankotarola) [inline src/client.ts:61]: should not be better to pass an object as payload? Thinking that the request payload could change and receive more than one field.

File: src/client.ts [hunk 2]
```diff
@@ -42,6 +48,60 @@ class SpaceClient {
...
+          }
+
+          resolve(res);
+        },
+      );
+    });
+  }
+
+  createBucket(
+    slug: string,
```
Comment (giankotarola) [inline src/client.ts:84]: same comment about the payload

File: example/index.js [hunk 3]
```diff
@@ -6,9 +6,33 @@ const client = new SpaceClient({
...
+  const txlStream = client.txlSubscribe();
+  txlStream.on('data', (res) => {
+    console.log(res.getBucket());
+  });
+
+  /**
+   * Open a File
+   */
+  const openFileRes = await client.openFile(entriesList[0].getPath());
+  console.log(openFileRes.getLocation());
```
Comment (giankotarola) [inline example/index.js:35]: is this not the same as the code on the readme?

File: src/client.ts (hunk 1 above)
Comment (gpuente) [reply src/client.ts]: @giankotarola  changed on b87d1b0

File: src/client.ts (hunk 2 above)
Comment (gpuente) [reply src/client.ts]: @giankotarola changed on b87d1b0

File: example/index.js (hunk 3 above)
Comment (gpuente) [reply example/index.js]: this is for testing with the daemon once BE release the proxy

**Pull request merged**